src/components/CollateralSurplusAction.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
+import { Button, Flex, Spinner } from '@chakra-ui/react'
+
+import { LiquityStoreState } from '@liquity/lib-base'
+import { useLiquitySelector } from '../hooks/useLiquitySelector'
```

Comment:
can we use absolute paths instead of relative imports?
```suggestion
import { useLiquitySelector } from 'src/components/hooks/useLiquitySelector'
```